quickjs.c

```diff
@@ -32561,7 +32560,7 @@ typedef enum BCTagEnum {
     BC_TAG_OBJECT_REFERENCE,
 } BCTagEnum;
 
-#define BC_VERSION 9
+#define BC_VERSION 10
```

Comment:
For my understanding, is the bump necessary because the atom ids change when the `#ifdef CONFIG_ATOMICS` is removed from quickjs-atom.h?

Reply:
Correct. That's the source of the bytecode incompatibility.